src/stanExports_estimate_infections.h

```diff
@@ -186,9 +186,9 @@ discretised_gamma_pmf(const int& y,
         stan::math::fill(beta, DUMMY_VAR__);
         stan::math::assign(beta,(mu / pow(c_sigma, 2)));
         current_statement_begin__ = 32;
-        stan::math::assign(alpha, (logical_lt(alpha, 0) ? stan::math::promote_scalar<local_scalar_t__>(1e-5) : stan::math::promote_scalar<local_scalar_t__>(alpha) ));
+        stan::math::assign(alpha, (logical_lte(alpha, 0) ? stan::math::promote_scalar<local_scalar_t__>(1e-5) : stan::math::promote_scalar<local_scalar_t__>(alpha) ));
```

Comment:
I can't see what's triggered this change, can I just get a sanity check to confirm it's intentional (even if generated)?
< to <=

Reply:
yes - recompiled the stan code from a previous PR built by another author (and not compiled). Fixes a marginal issue at the boundary.

Reply:
I dislike changes coming from other tasks but if it's good that's fine. I wonder if we could put a compile step on a github action to prevent this happening? (#lowpriority)